document-models/rwa-portfolio/rwa-portfolio.json

```diff
@@ -0,0 +1,256 @@
+                    "initialValue": "{\"principalLender\":\"id-1\",\"accounts\":[],\"spvs\":[],\"feeTypes\":[],\"fixedIncomeTypes\":[],\"attachments\":[],\"portfolio\":[],\"transactions\":[]}",
+                    "examples": []
+                },
+                "local": {
```

Comment:
local schema and initial state not included, see https://github.com/powerhouse-inc/document-model-electron/issues/66